lib/app/upload.ex

```diff
@@ -5,8 +5,8 @@ defmodule App.Upload do
   import SweetXml
   require Logger
 
-  @original_bucket Application.compile_env(:ex_aws, :original_bucket)
-  @compressed_bucket Application.compile_env(:ex_aws, :compressed_bucket)
+  @original_bucket Application.get_env(:ex_aws, :original_bucket)
+  @compressed_bucket Application.get_env(:ex_aws, :compressed_bucket)
```

Comment:
This will give us a few warnings but is acceptable for avoiding duplicating of definitions. 👍

Reply:
They will but since they're only used one time in each function, we can get away with not using these constants in the module body (even though it's cleaner).

Reply:
Fetching the envs from the environment on `each` invocation of the `upload` function will add latency ... ⏳ 
We can remove this when we eventually refactor to use `B2` #98 👍